pkg/executor/statefulset.go

```diff
@@ -16,17 +16,18 @@ package executor
 
 import (
 	"context"
+	"path/filepath"
+	"strings"
+
 	"github.com/go-logr/logr"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
-	"path/filepath"
-	"strings"
 
 	kdv1 "github.com/bluek8s/kubedirector/pkg/apis/kubedirector.bluedata.io/v1alpha1"
 	"github.com/bluek8s/kubedirector/pkg/catalog"
 	"github.com/bluek8s/kubedirector/pkg/shared"
 	appsv1 "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Can we make the tag corev1?

Reply:
Just bringing over the changes verbatim at the moment, unless we find functional problems.

I think this particular thing is something we need a more systematic solution to (if we want to enforce it), because several of us use IDEs that insert these import statements, and the IDEs are inclined to just alias this as "v1".